### PR TITLE
Add status to resque initd

### PR DESCRIPTION
**Pull request by NickLaMuro**
Also does the following:

* Drys up some duplicated code
* Removes unnecessary methods
* Renames methods to use less `do`
* Fixes some formatting

### CODE REVIEW

File: templates/default/resque-init.sh.erb [hunk 1]
```diff
@@ -61,32 +59,16 @@ do_start_one_do() {
...
   local pidfile
   local pid
   log_daemon_msg "--> Stopping Resque $1-$2"
 
   pidfile=$1/tmp/pids/$2.pid
   if [ -e $pidfile ]; then
-    pid=`cat $pidfile`
-    if [ "`ps -A -o pid= | grep -c $pid`" -eq 0 ]; then
+    if [ "$(pid_running $pidfile)" -eq 0 ]; then
       log_daemon_msg "---> Resque $1-$2 isn't running."
```
Comment (anfleene) [inline templates/default/resque-init.sh.erb]: I think this logic is backwards

File: templates/default/resque-init.sh.erb [hunk 2]
```diff
@@ -108,61 +90,47 @@ do_stop_one() {
...
+status_for_worker() {
+  local pidfile=$1/tmp/pids/$2.pid
 
-  pidfile=$1/tmp/pids/$2.pid
   if [ -e $pidfile ]; then
-    pid=`cat $pidfile`
-    log_daemon_msg "--> About to restart Resque $1-$2"
-    do_stop_one $1 $2 $3
-    do_start_one_do $1 $2 $3
+    if [ "$(pid_running $pidfile)" -eq 0 ]; then
```
Comment (anfleene) [inline templates/default/resque-init.sh.erb]: I think this logic is backwards too

Comment (NickLaMuro) [inline templates/default/resque-init.sh.erb]: First off, I didn't change the logic, so if it is wrong, than it has been wrong.

I also think it is correct.  Pid running returns a count of the number of pids that are currently matching the one passed in.  So if it is equal to zero, there isn't any pids that match.  What is incorrect is that zero usually means true in bash, which makes this a bit confusing.

I might change things so the `-eq 0` is part of the `pid_running` function, and then just use the result to make it a bit more clear and the logic more readable.

File: templates/default/resque-init.sh.erb (hunk 1 above)
Comment (NickLaMuro) [inline templates/default/resque-init.sh.erb]: See https://github.com/sportngin-cookbooks/resque/pull/5/files#r35110935

**Comment by anfleene** [comment]
These changes look good :+1:

**Pull request merged**